recipes/wk_spectra/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wk_spectra" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d217783761ae98ef5f2bfc83376d61bcb9d9e59cfbf650e29e21f0a986c996c9
+
+build:
```

Comment:
I think this can be `noarch: python`:

https://conda-forge.org/docs/meta.html#building-noarch-packages